examples/verify_tsqrtP_calibration.py

```diff
@@ -49,35 +49,30 @@
         fl.basename = f"amplitude = {amplitude:.1f}"
         if not d.get_units("t") == "s":
             psd.logger.info(
-                psd.strm(
-                    "units weren't set for the t axis or else I can't read them from the hdf5 file!"
-                )
+                "units weren't set for the t axis or else I can't read them from the hdf5 file!"
             )
             d.set_units("t", "s")
         d *= V_atten_ratio  # V at output of amplifier
         d /= np.sqrt(50)  # V/sqrt(R) = sqrt(P)
 
         # {{{ functions that streamline plotting the desired pulse length datasets
-        def switch_to_plot(d, thisj):
-            thislen = d.get_prop("programmed_t_pulse")[thisj] / 1e-6
+        def switch_to_plot(d, j):
+            thislen = d.get_prop("programmed_t_pulse")[j] / 1e-6
             fl.next(f"pulse length = {thislen:.2f} μs")
 
-        def indiv_plots(d, thislabel, thiscolor, thisj=None):
+        # PR this change vs. c772298 was not good  -- it's a bit forced, and as noted below, you're leaving out stuff that was done before.  So I rolled it back
+        def indiv_plots(d, thislabel, thiscolor):
```

Comment:
see (and remove) comment.  Please check that everything still works the way you want

Reply:
by "a bit force" I mean that you're switching from something fairly simple to something more complicated.  That's not bad, but it doesn't fulfill the end purpose (read the comment surrounding the code you're trying to replace below)

Reply:
(the PR comment refers to the entire function)